src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor({ health = 100, name }) {
+    this.health = health;
+    this.name = name;
+
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
```

Comment:
The constructor of the Herbivore class should pass the 'name' parameter to the super constructor as a string, not as an object. This will ensure that the 'name' property is correctly assigned.